libminifi/include/utils/ValueCaster.h

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenseas/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LIBMINIFI_INCLUDE_UTILS_VALUECASTER_H_
+#define LIBMINIFI_INCLUDE_UTILS_VALUECASTER_H_
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace utils {
+namespace internal {
+
+template<class T, class U>
+bool cast_if_in_range(T in, U& out) {
```

Comment:
I'm a bit conflicted about this as usually casts follow the `snake_case` style, but our code should follow the `camelCase` for function names.

Reply:
yeah, now I am conflicted too. I was under the impression that freefunctions should be snake_case (while member functions should be lowerCamelCase) but there is no documentation about this besides the mentioning the google style guide (which uses UpperCamelCase and sometimes snake_case)

I don't have a strong preference and I am comfortable with either snake_case or lowerCamelCase here. What is your preference?

Reply:
I don't have any strong preference here either, so you can keep it as it is.